src/data/content/community.yaml

```diff
@@ -507,67 +507,67 @@ components:
 
       cards:
 
-      - title: "Il modello di sito per le aziende sanitarie locali (ASL)"
+      - title: "Le iniziative nazionali per l'accessibilità"
         headingLevel: 4
         titleSmall: true
         cardEvent: true
-        img: /images/community/media-thumbnails/20240215-il-modello-di-sito-per-le-aziende-sanitarie-locali-asl.png
+        img: /images/community/media-thumbnails/20240516-le-iniziative-nazionali-per-l-accessibilita.jpg
```

Comment:
Questa immagine non la carica, ho controllato e nella directory corrispondente l'immagine non c'è, va caricata.

Reply:
La puoi controllare dal tab file changed sopra.

Reply:
Nella card della preview c'è, è solo il file og (social image) che non esiste ancora prima del merge 
<img width="1194" alt="image" src="https://github.com/italia/designers.italia.it/assets/6736103/fc27dc46-f919-47fe-ae20-1ced885fba88">

Reply:
Ok ora chiaro, grazie, non sapevo che funzionasse così!